src/App.js

```diff
@@ -63,36 +63,60 @@ export default function App() {
 
   return (
     <Appcontainer>
-      {currentPage === 'activity' && (
-        <DurationFilter onFilterDurationReset={filterDurationReset} onFilterDurationValue={setDurationFilterValue} />
-      )}
-      {currentPage === 'activity' && (
-        <TypeFilter onFilterTypeReset={filterTypeReset} onFilterTypeValue={setTypeFiltervalue} />
-      )}
-      {currentPage === 'activity' && (
-        <ActivityCards
-          activities={filteredActivities}
-          onSetActivities={setActivities}
-          onNavigate={page => setCurrentPage(page)}
-          onToggleFavorites={toggleFavorites}
-          onCloseConfirmationModal={closeModalConfirmationHandler}
-          onShowConfirmationHandler={showModalConfirmationHandler}
-          onDeleteActivity={deleteActivity}
-          showModalConfirmation={showModalConfirmation}
+      <Header />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <ActivityPage
+              activities={filteredActivities}
+              onSetActivities={setActivities}
+              onToggleFavorites={toggleFavorites}
+              onCloseConfirmationModal={closeModalConfirmationHandler}
+              onShowConfirmationHandler={showModalConfirmationHandler}
+              onDeleteActivity={deleteActivity}
+              showModalConfirmation={showModalConfirmation}
+              onFilterTypeReset={filterTypeReset}
+              onFilterTypeValue={setTypeFiltervalue}
+              onFilterDurationReset={filterDurationReset}
+              onFilterDurationValue={setDurationFilterValue}
```

Comment:
When reading this, I see three states that you pass down as props: `setActivities`, `setTypeFiltervalue` and `setDurationFilterValue`. Please don't pass states as props. I would _highly_ recommend to update the state in the component where the state is. In this case the `App.js`. 😉 

You already do this as well, with `deleteActivity` and `toggleFavorites`. 

EDIT:
You don't even need to pass the prop: `onSetActivities={setActivities}` here anymore. Please check which props are still used and needed and which are not!